app/controllers/bookmarks_controller.rb

```diff
@@ -9,7 +9,9 @@ def index
 
   def new
     @bookmark            = Bookmark.new
-    @bookmark.name       = params[:query].to_s.strip.split(/\s| = |!|~|>|</)[0]
+    query                = params[:query].to_s.squish
```

Comment:
In that case I would recommend to follow the one-issue per PR and treating
it as separate issue/PR

On Thu, Apr 7, 2016 at 1:50 PM, Ohad Levy <notifications@github.com> wrote:

> In app/controllers/bookmarks_controller.rb
> <https://github.com/theforeman/foreman/pull/3401#discussion_r58858322>:
>
> > @@ -9,7 +9,9 @@ def index
> >
> >    def new
> >      @bookmark            = Bookmark.new
> > -    @bookmark.name       = params[:query].to_s.strip.split(/\s| = |!|~|>|</)[0]
> > +    query                = params[:query].to_s.squish
>
> auto completer (complete_for) with mouse clicks and keyboard select
> (up/down etc) many times add excessive whitespaces, squish is equal to
> strip and ensure a whitespace exists only once. imho this looks better when
> saving the search string.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/3401/files/4b2b563608bb239d97ab7de7ff0dc876eea79bd0#r58858322>
>